Tasks/Common/ServiceFabricHelpers/Get-ServiceFabricEncryptedText.ps1

```diff
@@ -17,7 +17,8 @@ function Get-ServiceFabricEncryptedText
     $cert = Get-Item "Cert:\$defaultCertStoreLocation\$defaultCertStoreName\$serverCertThumbprint" -ErrorAction SilentlyContinue
     if (-not $cert)
     {
-        throw (Get-VstsLocString -Key ServerCertificateNotFoundForTextEncrypt -ArgumentList $serverCertThumbprint)
+        Write-Warning (Get-VstsLocString -Key ServerCertificateNotFoundForTextEncrypt -ArgumentList $serverCertThumbprint)
```

Comment:
why do you need to make this change, if you are not even making this call from the callers

Reply:
call will still go for secure clusters. this takes care that if certificate is not installed, then error is not thrown